src/components/Gallery/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import HeaderSection from '../HeaderSection'
+import Card from '../Card'
+import UsedButton from '../UsedButton'
+
+const Gallery = ({ items, headerText }) => {
+  return (
+    <section className="gallery">
+      <div className="gallery-container">
+        <div className="gallery-container__header">
+          <HeaderSection headerText={headerText} />
+          <UsedButton
```

Comment:
Try to use react-responsive library with mediaquery hook (I used this: const isTabletOrMobile = useMediaQuery({ query: "(max-width: 1200px)" }); ) and simple button component with if else statement. (isTabletOrMobile) ? Button primary.... : Button secondary